s3watcher/lambda/lib/Transfer.js

```diff
@@ -23,25 +22,24 @@ module.exports = {
         const upload = Upload.buildUpload(config, s3Event);
 
         const success = function() {
-            Logger.logDelete(config.stage, s3Object);
-
+            logger.log('Deleting from ingest bucket.', s3Object);
             return S3Helper.deleteS3Object(s3Object);
         };
 
         const failGraceful = function(e) {
-            Logger.logImportFail(config.stage, e);
-            Logger.logCopyToFailBucket(config.stage, s3CopyObject);
+            logger.log('Import failed.', e);
```

Comment:
Super minor and not as part of this PR but maybe you could (possibly...) `return this` in the `log` method in the logger module so that you can chain? You could then be all like 

```
logger
  .log('Import failed.', e)
  .log('Copying to fail bucket.', e);
```

Which is slightly nicer?